client/src/rpc_client.rs

```diff
@@ -1379,6 +1709,13 @@ impl RpcClient {
         self.supply_with_commitment(self.commitment())
     }
 
+    /// # RPC Reference
+    ///
+    /// This method corresponds directly to the [`getSupply`] RPC
+    /// method with the `Commitment` option set to `processed`.
```

Comment:
This doesn't seem right. This is what the example is doing, but not what the method does generally. Also, you generally haven't included parameter specifics in the RPC Reference sections, have you?

Reply:
Good catch. I've changed it.

I am not clear yet on exactly what I want to say in these rpc sections. For now I've just changed it to "this method corresponds directly to the getsupply rpc method".